korge/src/korlibs/korge/text/TextEditController.kt

```diff
@@ -24,19 +24,24 @@ import korlibs.time.*
 import kotlin.math.*
 import kotlin.text.isLetterOrDigit
 
+data class TextInputSettings(
+    // If true, render static non-blinking caret rather than blinking caret.
+    val useStaticCaret: Boolean = false
```

Comment:
I would rename it to blinking caret = true to name it in a positive way. 

Or alternatively, change it to configure the duration of the blinking:

```
val blinkingTime: Duration? = 1.seconds
```

I would go for the duration, that way we can use 0 or null for static non-blinking.
But as stated in the other comment, I believe this is a separate issue not related to the title of the PR "Support caret on empty text input"

Reply:
I'm willing to change the api but not willing to split into separate PRs as it's a PITA. I can just update the title to "Improve UITextInput" to make it more general.

Reply:
Works for me